daal4py/sklearn/decomposition/_pca.py

```diff
@@ -85,15 +85,15 @@ def _fit_full_daal4py(self, X, n_components):
             daal_n_components = n_components
 
         fpType = getFPType(X)
-        centering_algo = daal4py.normalization_zscore(
-            fptype=fpType, doScale=False)
+
+        covariance_algo = daal4py.covariance(fptype=fpType, outputMatrixType='covarianceMatrix')
```

Comment:
I thought normalization was happening on the side of the PCA? And what is called is covariance, not correlation by default. This is not true?

Reply:
Yes, SVD PCA uses normalization, but covariance/correlation PCA hasn't got normalization as a parameter.
And yes, this is covariance. I use it to be consistent with stock scikit-learn. But in C++ correlation is still default method